src/commands/filter-and-continue.yml

```diff
@@ -67,7 +67,12 @@ parameters:
       everything from the code-param-name-map.
     type: boolean
     default: false
+  additional-steps:
```

Comment:
do we want to call this pre-steps? or something like that. i dunno I'm still on the fence

Reply:
make sense show they come first